.eslintrc

```diff
@@ -14,6 +14,7 @@
     "import/no-named-as-default": 0,
     "import/no-named-as-default-member": 0,
     "react/prop-types": 1,
+    "react/jsx-props-no-spreading": 0,
```

Comment:
Wydaje mi się, że trzeba dodać jeszcze "camelcase": "off" bo bez tego nie mogłem wgl odpalić

Reply:
<img width="467" alt="Zrzut ekranu 2021-02-15 o 16 28 09" src="https://user-images.githubusercontent.com/48855027/107965423-ce3a9000-6faa-11eb-982d-21c521ba3413.png">
ja bym po prostu to zamienił na camelcase żeby wszędzie było tak samo